lambda.js

```diff
@@ -1,5 +1,4 @@
-"use strict";
-const AWS = require("aws-sdk");
+"use strict";;
```

Comment:
Interesting.  I should remove that extra semi-colon that came on after the migration commands.